shaders/src/common_lighting.fs

```diff
@@ -89,9 +151,9 @@ vec3 getReflectedVector(const PixelParams pixel, const vec3 v, const vec3 n) {
     float bendFactor          = abs(pixel.anisotropy) * saturate(5.0 * pixel.perceptualRoughness);
     vec3  bentNormal          = normalize(mix(n, anisotropicNormal, bendFactor));
 
-    vec3 r = reflect(-v, bentNormal);
+    vec3 r = GetAdjustedReflectedDirection(-v, bentNormal);
 #else
-    vec3 r = reflect(-v, n);
+    vec3 r = GetAdjustedReflectedDirection(-v, n);
```

Comment:
maybe update the comment of this function.

Reply:
Added some clarification in 6dc6210f556cee2f8ce36cfd404588c5ddb3fa2d .